Modules/Core/Common/include/itkColorTable.hxx

```diff
@@ -88,7 +88,10 @@ ColorTable<TPixel>::UseDiscreteColors()
   // set to NumericTraits<TPixel>::max().
   typename NumericTraits<TPixel>::RealType realMax(1.0 * scale + shift);
   TPixel                                   pixelMax(NumericTraits<TPixel>::max());
-  if (realMax < NumericTraits<TPixel>::max())
+  // Converting from TPixel to RealType may introduce a rounding error, so do static_cast
+  constexpr typename NumericTraits<TPixel>::RealType max_value_converted =
```

Comment:
Shouldn't this be `constexpr auto max_value_converted` due to explicit cast? Also repeated twice below.

Reply:
Fixed in upcomming rebase.